tests/appsec/iast/sink/test_hardcoded_secrets.py

```diff
@@ -2,27 +2,43 @@
 # This product includes software developed at Datadog (https://www.datadoghq.com/).
 # Copyright 2021 Datadog, Inc.
 
-from utils import coverage, missing_feature, features
-from .._test_iast_fixtures import BaseSinkTest, DetectionStage
+from utils import coverage, interfaces, weblog, features, context
 
 
 @features.iast_sink_hardcoded_secrets
 @coverage.basic
-class Test_HardcodedSecrets(BaseSinkTest):
+class Test_HardcodedSecrets:
```

Comment:
Could you add a small comment explaining why you did not inherit from `BaseSinkTest`? As it's the only one, it may help future readers.